data.sql

```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
 (SELECT ID FROM OWNERS WHERE FULL_NAME = 'Dean Winchester')
 WHERE NAME = 'Angemon'
 OR NAME = 'Boarmon'
+
+
+
+-- insert the following data for vets
+INSERT INTO VETS(
+  NAME, 
+  AGE,
+  DATE_OF_GRADUATION
+)
+ VALUES
+ (
+	 'Vet William Tatcher',
+	 45, 
+	 '2000-4-23'
+ 
+ );
+ 
+ 
+ INSERT INTO VETS(
+  NAME, 
+  AGE,
+  DATE_OF_GRADUATION
+)
+ VALUES
+ (
+	 'Vet Maisy Smith',
+	 26, 
+	 '2019-01-17'
+ 
+ );
+ 
+ 
+ 
+ INSERT INTO VETS(
+  NAME, 
+  AGE,
+  DATE_OF_GRADUATION
+)
+ VALUES
+ (
+	 'Vet Stephanie Mendez',
+	 64, 
+	 '1981-05-04'
+ 
+ );
+ 
+  INSERT INTO VETS(
+  NAME, 
+  AGE,
+  DATE_OF_GRADUATION
+)
+ VALUES
+ (
+	 'Vet Jack Harkness',
+	 38, 
+	 '2008-06-08'
+ 
+ );
+
+-- Insert the following data for specialties
+
+ INSERT INTO SPECIALIZATIONS (
+ VID,
+ SID
```

Comment:
- [ ] Please use descriptive column names. `vet_id` and `species_id` are recommended.
Giving descriptive names will right away let the reviewer or other developers using your database know exactly what you are referring to.

Please update it in all the areas where you use `VID` and `SID`